modules/importer/src/server/mod.rs

```diff
@@ -155,6 +157,37 @@ impl Server {
 
         Ok(Some(result))
     }
+
+    /// Reset all jobs back into non-running state.
+    ///
+    /// This is intended when the application starts up, to reset stale job states. Making it
+    /// possible to re-run them when they are due.
+    ///
+    /// **NOTE:** we can only do this as we're intended to be a single-process worker.
+    async fn reset_all_jobs(&self, service: &ImporterService) -> anyhow::Result<()> {
+        for importer in service.list().await? {
+            if importer.data.state == State::Running {
+                log::warn!(
```

Comment:
I still think WARN is overkill here. If everything I wanted to run before the crash is still gonna run after, why yell at me?

Reply:
Lowered it to `info`